libs/shared/src/types/builder/builder.types.ts

```diff
@@ -1,20 +1,43 @@
-export type BuilderGroupValues = 'AND' | 'OR';
+export enum FieldOperatorEnum {
+  ALL_IN = 'ALL_IN',
+  ANY_IN = 'ANY_IN',
+  BETWEEN = 'BETWEEN',
+  EQUAL = 'EQUAL',
+  IN = 'IN',
+  IS_DEFINED = 'IS_DEFINED',
+  LARGER = 'LARGER',
+  LARGER_EQUAL = 'LARGER_EQUAL',
+  LIKE = 'LIKE',
+  NOT_BETWEEN = 'NOT_BETWEEN',
+  NOT_EQUAL = 'NOT_EQUAL',
+  NOT_IN = 'NOT_IN',
+  NOT_LIKE = 'NOT_LIKE',
+  SMALLER = 'SMALLER',
+  SMALLER_EQUAL = 'SMALLER_EQUAL',
+}
```

Comment:
We might not be able to implement them all or just a subset because the nature of the payload for the events broadcast, but big part of the subset can be reused for filtering therefore the creation of this enum (besides cleaning the code).